src/open_inwoner/js/components/index.js

```diff
@@ -42,18 +42,29 @@ const elementWrappers = [
   // add more when needed
 ]
 
+// harden against multiple events
+// should not be needed but protects against unforeseen issues
+let activeElements = []
+
 function wrapComponentsOf(targetElement) {
+  console.debug(['wrapComponentsOf', targetElement])
   // apply the javascript component wrappers
   for (const [selector, callable] of elementWrappers) {
     for (const elt of htmx.findAll(targetElement, selector)) {
-      callable(elt)
-      console.debug(['htmx re-activated component on: ' + selector, elt])
+      if (activeElements.indexOf(elt) < 0) {
+        callable(elt)
+        activeElements.push(elt)
+        console.log(['htmx re-activated component on: ' + selector, elt])
+      } else {
+        console.debug([
+          'htmx skipped duplicate re-activation of component: ' + selector,
+          elt,
+        ])
+      }
     }
   }
+  // clean-up removed elements
+  activeElements = activeElements.filter((elt) => elt.isConnected)
 }
 
-htmx.onLoad(() => {
-  document.body.addEventListener('htmx:afterSwap', (event) => {
-    wrapComponentsOf(event.target)
-  })
-})
```

Comment:
I hooked into the wrong event before: apparently htmx onload is not a general ("htmx is ready") but already the event we wanted to use, eg: when a new element is added. In effect this meant that this kept adding event listeners and make a huge invisible mess.